ecf/rtofs/plots/jevs_rtofs_headline_plots.ecf

```diff
@@ -36,11 +36,12 @@ export COMPONENT=rtofs
 export RUN=headline
 
 # set up VDATE and COMIN and COMOUT
-export VDATE=$(date --date="2 days ago" +%Y%m%d)
+export VDATE=$(date --date="4 days ago" +%Y%m%d)
 
 export COMIN=${COMIN:-$(compath.py $envir/com/$NET/${evs_ver})}
```

Comment:
This line should be deleted. That construct is not useful in the ecf file since there is no processing upstream. This line should be in the job file, and it is already.

Reply:
Done.